experiments/probabilistic_domain_adaptation/livecell/common.py

```diff
@@ -55,9 +55,9 @@ def get_unet():
     return UNet2d(in_channels=1, out_channels=1, initial_features=64, final_activation="Sigmoid", depth=4)
 
 
-def load_model(model, ckpt, state="model_state", device=None):
-    state = torch.load(os.path.join(ckpt, "best.pt"))[state]
-    model.load_state_dict(state)
+def load_model(model, ckpt, get_model=get_unet, device=None):
```

Comment:
You can now use this function for loading the model, it takes care of the pytorch 2 things: https://github.com/constantinpape/torch-em/blob/main/torch_em/util/util.py#L204

Reply:
Updated in the recent [commit](https://github.com/constantinpape/torch-em/pull/116/commits/18f289e6d6d8067ed625d80be0107b01bf9ab7ac#diff-414c225623f833e81230091d9429fed78c30d5a5c3f6fd26238ff9587d59202cR126).